Intermittent test failure: test_features_view_pagination

Seen here: https://app.circleci.com/pipelines/github/mozilla/experimenter/63455/workflows/9b1bedbb-2b9c-4aa9-8be6-ac932679f5c0/jobs/382687


```py
_____________ TestNimbusFeaturesView.test_features_view_pagination _____________
[gw2] linux -- Python 3.13.6 /usr/local/bin/python3.13

self = <experimenter.nimbus_ui.tests.test_views.TestNimbusFeaturesView testMethod=test_features_view_pagination>

    def test_features_view_pagination(self):
        application = NimbusExperiment.Application.DESKTOP
        for num in range(6):
            NimbusExperimentFactory.create(
                name=f"Experiment {num}",
                application=application,
                feature_configs=[self.feature_configs["feature-desktop"]],
            )
    
        base_url = reverse("nimbus-ui-features")
        response = self.client.get(
            f"{base_url}?application={application.value}&feature_configs={self.feature_configs['feature-desktop'].id}"
        )
        self.assertEqual(response.status_code, 200)
        self.assertEqual(len(response.context["experiments_delivered"]), 5)
>       self.assertEqual(len(response.context["experiments_with_qa_status"]), 5)
E       AssertionError: 4 != 5

experimenter/nimbus_ui/tests/test_views.py:3452: AssertionError
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-6054)

--- a/experimenter/experimenter/nimbus_ui/tests/test_views.py
+++ b/experimenter/experimenter/nimbus_ui/tests/test_views.py
@@ -3441,6 +3441,7 @@ class TestNimbusFeaturesView(AuthTestCase):
                 name=f"Experiment {num}",
                 application=application,
                 feature_configs=[self.feature_configs["feature-desktop"]],
+                qa_status=NimbusExperiment.QAStatus.GREEN,
             )
 
         base_url = reverse("nimbus-ui-features")

